src/image/pixels.js

```diff
@@ -638,9 +638,9 @@ p5.prototype.set = function(x, y, imgOrCol) {
  * }
  *
  * function setup() {
- *   image(img, 0, 0);
+ *   image(img, 0, 0, width, height);
  *   let d = pixelDensity();
- *   let halfImage = 4 * (img.width * d) * (img.height * d / 2);
+ *   let halfImage = 4 * (width * d) * (height * d / 2);
```

Comment:
I think it's good to leave img.width and img.height here as it's a little more clear to understand how this is half the image.

Reply:
Thank you for suggesting the changes @lmccart ! I have a doubt, since we are explaining how to use
the pixels array of the main drawing canvas to update it, and are also using pixelDensity of the main canvas, 
I was thinking it would confuse the user which halfImage index we are trying to find (image or main drawing area). 

I would make the changes as your suggestions make it look less complex and clear to the user.
Please provide your thoughts on this, (sorry if I am wrong)
Thank you !

Reply:
@Ajayneethikannan ah I see, yes this is fine to leave as is since we are talking about half canvas image. thanks for explaiing.